products/BellHybrid/config/ProductConfig.in.hpp

```diff
@@ -3,4 +3,4 @@
 
 #pragma once
 
-#define ENABLE_GSM (0)
+#cmakedefine01 CONFIG_ENABLE_TEMP
```

Comment:
Is it a proper way to pass cmake-defined options?

Reply:
According to https://cmake.org/cmake/help/latest/command/configure_file.html, it is the preferred way of doing it instead of passing raw defines.